routers/flights.py

```
@@ -18,11 +18,11 @@ def get_db():
 @router.get("/")
 def read_flights(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
     flights = db.query(models.Flight).offset(skip).limit(limit).all()
-    return flights
+    return [{"flight_id": flight.flight_id, "flight_name": flight.flight_name} for flight in flights]
 
 @router.get("/{flight_id}")
 def read_flight(flight_id: int, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)